Budgets/Budgets.Domain.UnitTests/Budgets.Domain.UnitTests.csproj

```diff
@@ -7,6 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="FluentAssertions" Version="6.7.0" />
```

Comment:
Would be better in Budgets.Tests.Common so can be used in all test projects

Reply:
ok. I did not know that including the package only in a dependant referenced project would work